File: common/src/main/java/dev/wren/truss/TrussCommon.java
package dev.wren.truss;

import dev.wren.truss.platform.Services;

public class TrussCommon {

    public static final String ID = "truss";
    public static final String NAME = "Truss";

    public static void init() {
        Truss.LOGGER.info("common init for {} on {}", NAME, Services.PLATFORM.getPlatform());
    }
}

File: common/src/main/kotlin/dev/wren/truss/TrussCommon.kt
package dev.wren.truss

import com.mojang.brigadier.CommandDispatcher
import dev.wren.truss.commands.registerDebug
import net.minecraft.commands.CommandSourceStack
import net.minecraft.commands.Commands


object TrussCommon {

    fun init() {
        Truss.init()
        LOGGER.info("common init")
    }

    fun registerCommands(dispatcher: CommandDispatcher<CommandSourceStack>) {
        val root = Commands.literal("truss")
            .then(registerDebug())

        dispatcher.register(root)
    }

}